posts/app.py

```diff
@@ -60,28 +62,59 @@ def upload_new_post():
 @app.route('/v1/', methods=['GET'])
 def get_all_posts():
     """Get all posts for the whole event."""
-    # TODO(mukobi) change from fake data to real query from db
-    return jsonify({
-        "post_id": "abc123",
-        "event_id": "all",
-        "author_id": "mukobi",
-        "created_at": "2017-10-06T00:00:00+00:00",
-        "text": "This is the description of a post. The post is tagged with "
-                "the event id of 'all' and should be viewable by default.",
-        "images": [
-            "https://upload.wikimedia.org/wikipedia/commons/6/66/An_up-close_picture_of_a_curious_male_domestic_shorthair_tabby_cat.jpg",
-            "http://4.bp.blogspot.com/-w8U75TCuhgU/Tzw8TmaclvI/AAAAAAAABJ0/6fMMcRLAceM/s1600/Rabbit3.jpg"
-        ]})
+    # serialize otherwise nonserializable ObjectIDs
+    post_list = find_posts_in_db(POSTS_COLLECTION)
+    return json.loads(json_util.dumps(
```

Comment:
Consider changing this into a function so you don't have to repeat it

Reply:
Done, thanks for DRY reminder https://github.com/knative-portability/large-events/pull/24/commits/de1938ec530ea005e406717d092602c296daf6ea